src/services/iiif-parser.js

```diff
@@ -706,9 +706,11 @@ export function getStructureRanges(manifest, isPlaylist = false) {
       } else {
         hasRoot = true;
         // Total duration for all resources in the Manifest
-        manifestDuration = canvasesInfo.reduce(
-          (duration, canvas) => duration + canvas.range.end, 0
-        );
+        if (!isPlaylist) {
```

Comment:
Why is this check necessary now?  It looks like the conditional on line 693 should keep any playlists from reaching this code.

Reply:
Yes, I'm not sure why I put it there. Thanks for catching it.